test/data/outfitters.get.all.json

```diff
@@ -0,0 +1,65 @@
+{
+    "response": {
+        "success" : true,
+        "api": "FS ePermit API",
+        "type": "controller",
+        "verb": "get",
+        "src": "json",
+        "route": "permits/special-uses/commercial/outfitters"
+    },
+    "outfitters" : [
+      {
+        "accinstCn": "1095010356",
+        "region": 3,
+        "forest": 50552,
+        "district": 50552,
+        "authorizingOfficerName": "WILLIAM L.NOXON",
+        "authorizingOfficerTitle": null,
+        "applicant-info": {
+          "firstName": null,
+          "lastName": null,
+          "dayPhone": {
+            "areaCode": 541,
+            "number": 8156141,
+            "extension": 0,
+            "type": "BUSINESS"
+          },
+          "eveningPhone": {
+            "areaCode": 0,
+            "number": 0,
+            "extension": 0,
+            "type": null
+          },
+          "emailAddress": "test@email.org",
+          "mailingAddress": "ON ANW 0953",
+          "mailingCity": "ALBANY",
+          "mailingState": "OR",
+          "mailingZIP": 97321,
+          "organizationName": "COOLEY RIVER EXPEDITIONS",
+          "website": null,
+          "orgType": "Corporation"
+        },
+        "outfitters-fields":{
```

Comment:
Needs to be 'temp-outfitter-fields' not 'outfitters-fields' to match with schema.